usecase/usecase-webapi-xwalk-tests/samples/WebCL/res/COPYING

```diff
@@ -0,0 +1,3 @@
+All the file in this folder come from
+https://github.com/uebquite/webcl-demos/tree/master/Deforml
```

Comment:
Please use https://github.com/uebquite/webcl-demos/ with the commit, so that in future people can check the difference and make integration.